extension/src/NABfunctions.ts

```diff
@@ -105,6 +106,16 @@ export async function findNextUnTranslatedText() {
     console.log('Done: FindNextUnTranslatedText');
 }
 
+function removeCustomNotes(xlfUri: vscode.Uri) {
```

Comment:
Move to LanguageFunctions.ts? In this file we only have functions that are called from extension.ts.. (and maybe some helper functions for the UI...)

And maybe rename to removeCustomNotesFromFile?